constants.js

```diff
@@ -21,6 +21,8 @@ const HACKER_STATUSES = [
     HACKER_STATUS_CHECKED_IN
 ];
 const JOB_INTERESTS = ["Internship", "Full-time", "None"];
+// enum of type of requests
+const REQUEST_TYPES = ["GET", "POST", "PATCH", "DELETE"];
```

Comment:
Are these the only ones we use? Might be good to put down *all* of the http types:
https://www.w3schools.com/tags/ref_httpmethods.asp

Reply:
Talked about this with you